app/helpers/pages_helper.rb

```diff
@@ -217,21 +217,25 @@ def page_object(page)
     base.merge(layouts_and_plugins)
   end
 
-  def countries
-    ISO3166::Country.all.map do |c|
-      {
-        name: c.name,
-        alpha2: c.alpha2,
-        alpha3: c.alpha3,
-        country_code: c.country_code,
-        currency_code: c.currency_code,
-        eu_member: c.in_eu?,
-        eea_member: c.in_eea?
-      }
-    end
-  end
```

Comment:
This was removed because we're only using it for the `eea_member` property (for GDPR), and came with a performance hit (and about 200kb extra page size). It was easier to simply hardcode the countries in a separate file.